libs/platform/src/lib/components/form/platform-file-uploader/platform-file-uploader.component.ts

```diff
@@ -92,11 +99,13 @@ export class PlatformFileUploaderComponent extends CollectionBaseInput {
      * The model backing of the component. */
     changedFiles: SelectionModel<File>;
 
+    /**sets value file data*/
+    @Input()
     get value(): any {
-        throw new Error('Method not implemented.');
+        return this.files;
     }
     set value(value: any) {
-        throw new Error('Method not implemented.');
+        this.files = value;
```

Comment:
This class should be extending from `BaseInput` right? Any reason why `CollectionBaseInput` was specifically chosen? 
Also, `super.setValue()` and `super.getValue()` should be called.Other methods like `writeValue()` and others might also be needed. You can refer other form components implementation.